docs/tutorials/excel-power-automate-manual.md

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Run Office Scripts in Excel on the web with Power Automate'
+description: 'A tutorial about integrating Power Automate with Office Scripts using a manual trigger.'
+ms.date: 01/21/2020
+localization_priority: Normal
+---
+
+# Run Office Scripts in Excel on the web with Power Automate
+
+This tutorial will teach you how to run an Office Script for Excel on the web through [Power Automate](https://flow.microsoft.com).
+
+## Prerequisites
+
+Before starting this tutorial, you'll need access to Office Scripts, which requires the following:
+
+- [Excel on the web](https://www.office.com/launch/excel).
+- Ask your administrator to [enable Office Scripts for your organization](https://support.office.com/article/office-scripts-settings-in-m365-19d3c51a-6ca2-40ab-978d-60fa49554dcf), which adds the **Automate** tab to the ribbon.
+- [Preview access to Power Automate](https://us.tip1.flow.microsoft.com).
+
+> [!IMPORTANT]
+> This tutorial assumes you have completed the [Record, edit, and create Office Scripts in Excel on the web](excel-tutorial.md) tutorial.
+
+## Prepare the workbook
+
+We need to set up a workbook with specific workbook and worksheet names. This is because Power Automate cannot use relative Office Script APIs, such as `Workbook.getActiveWorksheet`.
+
+1. Create a new workbook named **MyWorkbook**.
+
+2. In the **MyWorkbook** workbook, create a worksheet called **TutorialWorksheet**.
+
+## Create an Office Script
+
+1. Go to the **Automate** tab and select **Code Editor**.
+
+2. Select **New Script**.
+
+3. Replace the default script with the following script. This script adds the current date and time to the first two cells of the **TutorialWorksheet** worksheet.
+
+    ```TypeScript
+    async function main(context: Excel.RequestContext) {
+      // Get the "TutorialWorksheet" worksheet from the workbook.
+      let worksheet = context.workbook.worksheets.getItem("TutorialWorksheet")
+
+      // Get the cells at A1 and B1.
+      let dateRange = worksheet.getRange("A1");
+      let timeRange = worksheet.getRange("B1");
+
+      // Get the current date and time using the Date APIs.
+      let date = new Date(Date.now());
+
+      // Add the date string to A1
+      dateRange.values = [[date.toLocaleDateString()]];
+
+      // Add the time string to B1
+      timeRange.values = [[date.toLocaleTimeString()]];
+    }
+    ```
+
+4. Rename the script to **Set date and time**. Press the script name to change it.
+
+5. Save the script by pressing **Save Script**.
+
+## Create an automated workflow with Power Automate
+
+1. Sign in to [Power Automate](https://us.tip1.flow.microsoft.com). You'll be taken to the main page, with your Flow actions displayed in the left-hand menu.
```

Comment:
`https://us.tip1.flow.microsoft.com` seems like an odd URL (and seems hardcoded to reference a server in the U.S. region) -- are we sure that's the right one to use?